app/components/gh-upgrade-notification.js

```diff
@@ -0,0 +1,8 @@
+import Ember from 'ember';
```

Comment:
@vkandy I know this isn't past WIP yet, but with all of the components can you please change them to use destructured ember properties through [ember-cli-shims](https://github.com/ember-cli/ember-cli-shims)?

The end result would look something like this:

```js
import Component from 'ember-component';

export default Component.extend({
// ....
```

Do that and it'll fix the linting issues 😄